README.md

```diff
@@ -18,15 +18,20 @@ Read the [overview](https://kueue.sigs.k8s.io/docs/overview/) to learn more.
-- **System insight:** Build-in [prometheus metrics](https://kueue.sigs.k8s.io/docs/reference/metrics/) to help monitor the state of the system, as well as Conditions.
+- **Integrations:** Built-in support for popular jobs, e.g. [BatchJob](https://kueue.sigs.k8s.io/docs/tasks/run/jobs/), [Kubeflow training jobs](https://kueue.sigs.k8s.io/docs/tasks/run/kubeflow/), [RayJob](https://kueue.sigs.k8s.io/docs/tasks/run/rayjobs/), [RayCluster](https://kueue.sigs.k8s.io/docs/tasks/run/rayclusters/), [JobSet](https://kueue.sigs.k8s.io/docs/tasks/run/jobsets/),  [plain Pod and Pod Groups](https://kueue.sigs.k8s.io/docs/tasks/run/plain_pods/).
+- **System insight:** Build-in [prometheus metrics](https://kueue.sigs.k8s.io/docs/reference/metrics/) to help monitor the state of the system, and on-demand visibility endpoint for [monitoring of pending workloads](https://kueue.sigs.k8s.io/docs/tasks/manage/monitor_pending_workloads/pending_workloads_on_demand/).
 - **AdmissionChecks:** A mechanism for internal or external components to influence whether a workload can be [admitted](https://kueue.sigs.k8s.io/docs/concepts/admission_check/).
 - **Advanced autoscaling support:** Integration with cluster-autoscaler's [provisioningRequest](https://kueue.sigs.k8s.io/docs/admission-check-controllers/provisioning/#job-using-a-provisioningrequest) via admissionChecks.
 - **All-or-nothing with ready Pods:** A timeout-based implementation of [All-or-nothing scheduling](https://kueue.sigs.k8s.io/docs/tasks/manage/setup_wait_for_pods_ready/).
 - **Partial admission:** Allows jobs to run with a [smaller parallelism](https://kueue.sigs.k8s.io/docs/tasks/run/jobs/#partial-admission), based on available quota, if the application supports it.
+- **Mixing training and inference**: Simultaneous management of batch workloads along with serving workloads (such as [Deployments](https://kueue.sigs.k8s.io/docs/tasks/run/deployment/) or [StatefulSets](https://kueue.sigs.k8s.io/docs/tasks/run/statefulset/))
+- **Configurable resource abstractions**: Allows to define quota in derived units, such as monetary credits
+- **Multi-cluster job dispatching:** called [MultiKueue](https://kueue.sigs.k8s.io/docs/concepts/multikueue/), allows to search for capacity and off-load the main cluster.
+- **Topology-Aware Scheduling**: Allows to optimize the pod-pod communication throughput by [scheduling aware of the data-center topology](https://kueue.sigs.k8s.io/docs/concepts/topology_aware_scheduling/).
+- **Managing Kueue-agnostic workloads**: [setting up a default Queue](https://kueue.sigs.k8s.io/docs/tasks/manage/enforce_job_management/setup_default_local_queue/) in a namespace allows managing all workloads created in that namespace without any Kueue-specific labels.
```

Comment:
I would like to minimize this section and focus only on generic AI workload problems. So, I would propose removing the "Configurable resource abstractions" and "Managing Kueue-agonostic workloads".

Because those features are the features that resolve Kueue problems for the AI workloads, and not generic problems in AI workloads worlds.

Reply:
I see, the section is meant to attract new users who have issues running their workloads. the 2 features are mostly solving issues of users who already adopting kueue, so I'm good with that. are you ok @dgrove-oss and @@yaroslava-serdiuk?

Reply:
I am fine with removing both Configurable resource abstractions" and "Managing Kueue-agonostic workloads" from this list.  They are useful features, but probably at the next level of detail.

Reply:
I synced with @yaroslava-serdiuk and she is good with that. Updated, and TBH I would like go further with minimizing this section because IMO it looks noisy still from the perspective of a potential future user.

I propose more tweaks in https://github.com/kubernetes-sigs/kueue/pull/3879/commits/5d938053c7c845f919b2f8692133e80e9a5044a5 - PTAL, basically:

In particular my candidates to drop:
- combine Partial Admission and Dynamic resource reclaim because they are conceptually very close
- merge Resource flavor fungibility with Resource management

WDYT @tenzen-y @dgrove-oss ?

Reply:
Yes, I'm ok with minimising the list of features. However from the page it not obvious that this is a subset of features. So it would be nice to add the link to the place with full list of features

Reply:
Right, note taken, but it will be a separate PR / issue. 

The complication is that we need to think about a good place in the docs, and think what would be the source of truth - because this does not seem to be list of KEPs, not the list of feature gates, maybe some mix of the two, but it is not clear to me yet how to approach it yet.